target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -36,10 +37,30 @@
 
 	soc {
 		pinmux@800000 {
-			i2c4_pins: i2c4_pinmux {
-				pins = "gpio12", "gpio13";
-				function = "gsbi4";
-				bias-disable;
```

Comment:
By the way, instead of removing i2c4 pins I think it's better to put a pinctrl into RPM node, as it's made in R7800 DTS https://github.com/dissent1/r7800/blob/sta14/target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts#L430
This way it's used in all close to Qualcomm repos (chromium, code aurora)